### PR TITLE
Cycloidal Drive REV-1

### PR DESCRIPTION
**Pull request by wraftus**
Disk Bearing: https://www.mcmaster.com/5972K159/
Pin Bushing: https://www.mcmaster.com/6294K444/

### CODE REVIEW

File: q-pup/motor_module/equations.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿"N"= 21
```
Comment (wmmc88) [inline q-pup/motor_module/equations.txt:1]: Is there the 40:1 variant too?

**Review by wmmc88 — COMMENTED** [review]
maybe add readme that has a table with PartName.SLDPRT <-> McMaster Carr Part Link mapping.

Also screenshots would be nice :P

**Comment by matthew-reynolds** [comment]
> maybe add readme that has a table with PartName.SLDPRT <-> McMaster Carr Part Link mapping.

The sldprt name is the same as the McMaster Carr part number

File: q-pup/motor_module/equations.txt (hunk 1 above)
Comment (matthew-reynolds) [reply q-pup/motor_module/equations.txt:1]: To be done in follow-up PR. :eyes: @wraftus

**Comment by matthew-reynolds** [comment]
Follow up:  Shrink bearing, leveraging through-hole for main motor shaft probably :eyes: @wraftus

**Review by matthew-reynolds — APPROVED** [review]

**Pull request merged**